ProjectLighthouse/PlayerData/Profiles/UserUpdate.cs

```diff
@@ -18,6 +18,9 @@ public class UserUpdate
     [XmlElement("planets")]
     public string? PlanetHash { get; set; }
 
+    [XmlElement("crossControlPlanet")]
+    public string? LBP2CrossControlPlanetHash { get; set; }
```

Comment:
This name is definitely too long, I think it would be better if you dropped the `LBP2` at the beginning or renamed it to `PlanetHashCrossControl` or `PlanetHashCC` like the user